src/config/mod.rs

```diff
@@ -0,0 +1,199 @@
+mod cli_options;
+mod yml_parser;
+
+use core::panic;
```

Comment:
I believe `panic` is already in the prelude 🤔

Reply:
The linter forgot to bark at me. :sweat_smile: Good catch!